### DISCUSSION TITLE
Filtering projects

### DISCUSSION THREAD

**Discussion by semmet95** [protontypes/open-sustainable-technology#770] [Q&A] [answered]
Hey everyone.
Firstly, thanks for this great initiative. It's good to see this huge list trying to use tech for good causes.

I was wondering if there is any Slack channel or any other resource where I could check which programs are actively under development, or which ones need help.
I checked out some repos, but they seem inactive. Additionally, it would be great if there was any way to filter this list to focus only on the projects using tech stacks I'm familiar with, or any other filtering use case for that matter.

Thank you :)

**Comment by Ly0n** [comment, marked as answer]
@semmet95 Thanks for your interest in the project. We developed exactly what you are looking for: 
https://climatetriage.com/

We do not have a Slack channel at the moment but we regularly meet online. You can find the calender here: 
https://opensustain.tech/contributing/#join-the-community
